app/assets/javascripts/backbone/views/loan_tab_view.coffee

```diff
@@ -0,0 +1,17 @@
+class MS.Views.LoanTabView extends Backbone.View
+
+  initialize: (params) ->
+    @calEvents = params.calEvents
+    calendarTab = @$('.calendar-tab').closest('li')
+    @openCalendar() if @$(calendarTab).hasClass('active')
+
+  events: ->
+    'shown.bs.tab .calendar-tab': 'openCalendar'
+
+  openCalendar: (e) ->
+    if !@calView
```

Comment:
Generally if you have `if !something ... else other` you should switch the if and else and get rid of the !.